### DISCUSSION TITLE
NS1 is hiring a NetBox developer!

### DISCUSSION THREAD

**Discussion by jeremystretch** [netbox-community/netbox#9297] [General]
Hey folks! I wanted to share that we're looking to hire a Sr. Software Engineer focused on NetBox development at NS1! This is a full-time employment opportunity available for remote work out of the US, UK, or Ireland.

This role will work closely with myself and the rest of the NetBox team here at NS1 to build both our NetBox Cloud SaaS offering as well as contribute to the open source NetBox project. We're looking for someone with a wealth of Python and Django experience, ideally with a focus toward networking and network automation. And of course expertise with NetBox would be ideal.

A bit more detail about what the role entails:

* Help lead the development of NetBox Cloud’s roadmap.
* Engage in product and roadmap planning discussions.
* Contribute to documentation and supporting tooling for both the NetBox community and NS1’s NetBox offerings.
* Work closely with NS1’s NetBox-focused SREs.
* Support the open source NetBox project and assist in delivering its roadmap.
* Seek new ways to deliver the best possible user experience for NetBox users.
* Become an active member of the NetBox open source community.

This is a fantastic opportunity to work full-time on NetBox and related integrations, and to contribute directly to software that powers network automation for thousands of organizations globally. To learn more about this role, please check out the [official posting at NS1](https://ns1.com/careers?gh_jid=4186206). You're also welcome to contact me directly on the [NetDev Slack](https://netdev.chat/).

**Comment by bobi-saputra** [comment]
im interested
